src/pages/components/progress-indicator/usage.mdx

```diff
@@ -45,41 +48,208 @@ set of steps, guiding toward the completion of a specified process.
+| Completed   | A step is complete when a user has filled out the required information within a step and progressed to the following step. When possible, use validation to confirm that a step has been completed before the user continues. All steps that have been completed are indicated by an outlined circle with a checkmark and a blue active line. |
+| Current     | A step is current when a user is interacting with the information within that step. The current step the user is on is indicated by a half-filled circle and a blue active line.                                                                                                                                                              |
+| Not started | A step is not started when a user has not yet interacted with that step. Steps the user has not encountered yet, or future steps, are indicated by an outlined circle and a gray active line.                                                                                                                                                 |
+| Error       | A step may be in error when a user has entered invalid or incomplete information. There could also be a server-side error. Provide clear information about the error and guidance on how to resolve the issue.                                                                                                                                |
+| Disabled    | A step is disabled when all interactive functions have been removed. Unlike read-only states, disabled states are not focusable, are not read by screen readers, and do not need to pass visual contrast, making them inaccessible if they need to be interpreted.                                                                            |
+| Hover       | Use hover state when a user's mouse cursor is hovering over the progress indicator's step.                                                                                                                                                                                                                                                    |
+| Focus       | When a user tabs to or clicks on the progress indicator's step, the step becomes focused, indicating the user has successfully navigated to the component.                                                                                                                                                                                    |
+
+### Interactions
+
+#### Mouse
+
+Users can trigger a state change by clicking anywhere in the step progress area.
+
+#### Keyboard
+
+One tab should be selected by default. Users can navigate between tabs by
+pressing right or left arrow keys. For additional keyboard interactions, see the
+[Accessibility](/components/progress-indicator/accessibility/) tab.
+
+#### Screen readers
```

Comment:
Check in with @mbgower on this if we want this info here anymore.

Reply:
Usually in his reviews he says to remove this "Screen readers" section. This section was created a long time ago in our docs.

Reply:
Thanks resolving this in my next commit.